src/main/java/rpgcharacters/userFlow/InitMenu.java

```diff
@@ -0,0 +1,68 @@
+package rpgcharacters.userflow;
+import java.util.Scanner;
+import java.util.InputMismatchException;
+
+public class InitMenu implements Menu {
+
+    private Scanner sc;
+
+    /**
+    * Constructor Method
+    */
+    public InitMenu () {
+        this.sc = new Scanner(System.in);
+    }
+
+    private void printMenuTitle() {
+        System.out.println("\n-------------------------------------------------------");
+        System.out.println("Init Menu");
+        System.out.println("-------------------------------------------------------");
+    }
+
+    private void printOptions () {
+        System.out.println(
+            "Available options:\n" +
+            "\t1: Login\n" +
+            "\t2: Create User\n" +
+            "\t3: Exit\n" +
+            "-------------------------------------------------------" // 50 chars
+        );
+        System.out.print("Please enter the number of the desired option here: ");
+    }
+
+    /**
+    * Defines the loop for this menu
+    */
+    public void enter () {
+        printMenuTitle();
+        int input;
```

Comment:
`input` needs to be set to something in all 5 files with a `try`, maybe set to 0?

`mvn package` succeeds but I get this error when trying to run the jar:
```
Exception in thread "main" java.lang.Error: Unresolved compilation problem: 
	The local variable input may not have been initialized

	at rpgcharacters.userflow.InitMenu.enter(InitMenu.java:65)
	at rpgcharacters.RpgCharacters.main(RpgCharacters.java:57)
```